drivers/sensor/bmp388/bmp388.c

```diff
@@ -207,6 +207,102 @@ int bmp388_reg_field_update(const struct device *dev,
 	return cfg->ops->reg_field_update(dev, reg, mask, val);
 }
 
+#if defined(CONFIG_BMP388_VALIDATE_TIMINGS)
+static uint32_t bmp388_pow(uint8_t base, uint8_t power)
```

Comment:
This commit needs a code formatting pass to conform to standard and match rest of the file.

Reply:
Oh... actually I intentionally avoided that because apparently the file was never formatted according to the `clang-format` config, so clang was adding a LOT of modifications, which I thought would make reviewing messier.
I can add a formatting commit at the end, so it stays cleanly separate?

Reply:
for reference, this is the result of running `clang-format -i drivers/sensor/bmp388/bmp388*`:

```
 drivers/sensor/bmp388/bmp388.c         | 410 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++----------------------------------------------------------------------------------------------------
 drivers/sensor/bmp388/bmp388.h         | 108 +++++++++++++++++++-------------------------
 drivers/sensor/bmp388/bmp388_trigger.c |  55 +++++++----------------
 3 files changed, 222 insertions(+), 351 deletions(-)
 ```

Ok to go ahead and add it to the PR?

Reply:
Maybe add it as a separate commit inside this PR? At least match the existing formatting in the file. ;-)

Reply:
Will do.